frontend/src/main/java/puf/frisbee/frontend/viewmodel/GameViewModel.java

```diff
@@ -176,35 +179,50 @@ public void stopCharacterMoveRight(String character) {
 	private boolean isCharacterLeftAllowedToJump() {
 		return this.characterLeftYPosition.getValue() == levelModel.getInitialCharacterYPosition()
 				&& !this.isCharacterLeftMovingLeft
-				&& !this.isCharacterLeftMovingRight;
+				&& !this.isCharacterLeftMovingRight
```

Comment:
Das werden jetzt soviele ifs, die könnte man in ein `isCharacterAllowedToJump` zusammenfassen... wäre dann vllt lesbarer (wenn auch etwas verschachtelter) 🤔